src/cli/codes.rs

```diff
@@ -8,27 +8,38 @@ pub async fn attach_pubkey_to_code(
     code_id: Uuid,
     config: &Config,
 ) -> std::io::Result<()> {
-    if let Ok(p) = RistrettoPublicKey::from_hex(&pubkey_hex) {
-        let client = actix_web::client::Client::new();
-        match client
-            .post(format!(
+    match RistrettoPublicKey::from_hex(&pubkey_hex) {
+        Ok(p) => {
+            let client = actix_web::client::Client::new();
+            println!(
```

Comment:
did you intend to leave this println in?

Reply:
I did not! :eyes: Thanks!